utils/sccli.py

```diff
@@ -120,12 +120,16 @@ def service_status(service_name):
         services = ['etcd', 'kube-apiserver', 'kube-controller-manager',
                     'kube-scheduler', 'kube-proxy', 'kubelet']
         for service in services:
+            while system("systemctl is-active %s" % service)[0].strip() == 'activating':
+                pass
```

Comment:
`pass` is a very CPU intensive operation as it is null operation. So it will continuously run `systemctl is-active %s" % service)` till it comes out of the loop.  If you do 
```
while True:
   pass
```
It takes 100% CPU. time.sleep(1) is a better option and less CPU intensive.